sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/LogsQueryOptions.java

```diff
@@ -73,6 +75,25 @@ public LogsQueryOptions setIncludeStatistics(boolean includeStatistics) {
         return this;
     }
 
+    /**
+     * If a query has partial errors, the result is returned instead of throwing an exception if this is set to true.
+     * The partial error information is available as part of the query result.
+     *
+     * @return Returns true if partial errors should not throw exception.
+     */
+    public boolean isDisableExceptionOnPartialErrors() {
```

Comment:
This seems like an odd name. I understand we try to use is and set for booleans but somehow hasDisabledExceptionOnPartialErrors has a better ring to it.

Reply:
After discussion with the v-team, renamed the property name to allowPartialErrors and we generally use `is` prefix for boolean getters just to be consistent.